### ISSUE TITLE
Failed to load API definition.

### ISSUE THREAD

**Issue by jasiqli** [cloudflare/itty-router-openapi#59]
If the main router has `base` set, going to `/<base>/docs` renders

![failed-to-load-api-def](https://github.com/cloudflare/itty-router-openapi/assets/107296949/fd232773-99b7-4f58-b1d4-29def237da84)

and `/<base>/redocs` is broken too. Removal of the `base` configuration renders the pages.

The `openapi.json` file does load directly at `/<base>/openapi.json` though.

Using the [`workers-sdk` openapi example](https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-openapi)

```javascript
import { OpenAPIRouter } from '@cloudflare/itty-router-openapi';
import { TaskCreate, TaskDelete, TaskFetch, TaskList } from './tasks';

const router = OpenAPIRouter({
  base: "/tasker",
  openapi_url: '/tasker/openapi.json',
  schema: {
    info: {
      title: 'Worker OpenAPI Example',
      version: '1.0',
    },
  },
});

router.get('/api/tasks/', TaskList);
router.post('/api/tasks/', TaskCreate);
router.get('/api/tasks/:taskSlug/', TaskFetch);
router.delete('/api/tasks/:taskSlug/', TaskDelete);

// Redirect root request to the /docs page
router.original.get('/', request => Response.redirect(`${request.url}/docs`, 302));
    // Added `/` above or it redirects to `/<base>docs` not `/<base>/docs`
    
// 404 for everything else
router.all('*', () => new Response('Not Found.', { status: 404 }));

export default {
  fetch: router.handle,
};
```

All other routes continue to work when `base` is set.

![routes-work](https://github.com/cloudflare/itty-router-openapi/assets/107296949/3539214e-420d-4555-8065-634c36155356)

Setting `docs_url` or `openapi_url` didn't change the above result. Tested with `0.1.0` -> `0.1.6`.

**Comment by G4brym** [comment]
Hey @jasiqli  i've published a new version (v0.1.10) that fixes this issue

**Comment by jasiqli** [comment]
This works just as I hoped now @G4brym. My *(possibly completely inane)* demonstration *(and reason for this report in the first place)* is https://github.com/jasiqli/pages-openapi Any thoughts appreciated.
